README.md

```diff
@@ -13,29 +12,42 @@ An email app for [ownCloud](https://owncloud.org) (minimum version 8). At the mo
 We’re working towards a [0.1 release](https://github.com/owncloud/mail/milestones/0.1) at the moment. If you experience any issues or have enhancement suggestions, please report them in our [issue tracker](https://github.com/owncloud/mail/issues).
 
 
-Maintainers
------------
-- [Thomas Müller](https://github.com/DeepDiver1975)
-- [Jan-Christoph Borchardt](https://github.com/jancborchardt)
-- [Lukas Reschke](https://github.com/LukasReschke)
-- [Christoph Wurst](https://github.com/wurstchristoph)
-- [Thomas Imbreckx](https://github.com/zinks-)
-- [Plato Leung](https://github.com/PoPoutdoor)
-- [Steffen Lindner](https://github.com/Gomez)
-- past contributors: [Jakob Sack](https://github.com/jakobsack), [Bart Visscher](https://github.com/bartv2), [Sebastian Schmid](https://github.com/sebastian-schmid)
+## Why is this so awesome?
 
-If you’d like to join, just go through the [issue list](https://github.com/owncloud/mail/issues) and fix some. :)
+* :rocket: **Integration with other ownCloud apps!** Currently Contacts & Files – more to come.
+* :+1: **You can use multiple mail accounts!** Unlike [Roundcube](https://roundcube.net) for example.
```

Comment:
Perhaps let's avoid naming and shaming :smile: Best would be a straightforward 'Gmail and personal account? No problem!', but if you must, then 'unlike some other webmail solutions'